sdk/cosmosdb/cosmos/src/client/ChangeFeed/ChangeFeedForEpkRange.ts

```diff
@@ -419,6 +419,10 @@ export class ChangeFeedForEpkRange<T> implements ChangeFeedPullModelIterator<T>
       feedOptions.sessionToken = this.changeFeedOptions.sessionToken;
     }
 
+    if (this.changeFeedOptions.excludedLocations) {
```

Comment:
DO we need this check if eedOptions.excludedLocations is optional?

Reply:
All the fields like `sessionToken` in `changeFeedOptions` are optional. There is check for each field.
https://github.com/Azure/azure-sdk-for-js/blob/7085233353c7806529ab4c9771e3e5a0443ff8ac/sdk/cosmosdb/cosmos/src/client/ChangeFeed/InternalChangeFeedOptions.ts#L10